types/yargs/ts4.1/yargs-tests.ts

```diff
@@ -123,8 +123,7 @@ async function Argv$parsing() {
     const argv1 = await yargs.parse();
     const argv2 = yargs(['-x', '1', '-y', '2']).parseSync();
     const argv3 = yargs.parseSync(['-x', '1', '-y', '2']);
-    const argv4 = await yargs();
```

Comment:
AFAIK, `yargs()` does not return a promise

Reply:
Also, if `argv4` were kept, it should be the following:
```js
const argv4 = yargs().argv
```

The result of`yargs()` does not return `argv` -- it has `argv` as a property

Reply:
[parseAsync](http://yargs.js.org/docs/#api-reference-parseasyncargs-context-parsecallback) makes sense to use the `await` keyword
```js
const argv4 = await yargs().parseAsync()
```